### DISCUSSION TITLE
Unable to add OS integration due to failing config

### DISCUSSION THREAD

**Discussion by slayer531** [vinteo/hass-opensprinkler#170] [Q&A] [answered]
Hi there,

I've gone through HACS, and trying to configure the integration, but keep getting the attached error
![image](https://user-images.githubusercontent.com/13462862/163800179-8c1f1a65-d3fe-4f64-9072-cb245ed5894d.png)

I'm running an old version of OP
![image](https://user-images.githubusercontent.com/13462862/163800242-d02344d7-a6aa-4d87-b1ce-6db0731d10df.png)

Tried using the etho and wlan mac address, no joy. (the raspberry pi is connected via wifi)
Using the same url (up until the port number) and password as I use in a browser.

Any help will be appreciated.

**Comment by vinteo** [comment, marked as answer]
Sorry only 2.1.5 and higher is supported
